specification.md

```diff
@@ -0,0 +1,54 @@
+# RFC Websocket Specification Proposal - 0.0.1
+
+> A websocket only specification.
+
+# TOC
+- [General](#general)
+- [Authentication](#Authentication)
+- [Event base type](#event-base-type)
+- [API](./api/)
+    - [Party API](./api/party-api.md)
+    - [Game API](./api/game-api.md)
+    - [Chat API](./api/chat-api.md)
+    - [Errors](./errors.md)
+- [Workflows](./usecases/wip.md)
+- [TODOs](#todos)
+
+
+# General
+The API is versioned by the Websocket path by the major version identifier with the prefix `ws/v`. Meaning, since this is Version `0.0.1` the major verion of the specification is 0. Resulting in a websocket path `ws/v0`. This means a valid websocket url for this api version would be something along the lines `ws://super-host.dev/ws/v0`.
+
+# Authentication
+To make sure no user can immitate another user, specific actions are secured by a simple "authentication". This authentication is not permanant and should not require an account or anything else besides a username. To acomplish this we use [JWT](https://jwt.io/introduction). A Token is created when a user joins a party and is used to authenticate chat messages and game actions. The API Calls, which should be checked by the server are marked with `Authentication needed`. 
```

Comment:
The details of the JWT are not relevant. You dont need to know them to read the claims. But since the only claim in the token is the partyId and username which are entered by the user, the frontend can treat the token the same way it would treat an opaque token or a session id.